README.md

```diff
@@ -122,78 +122,6 @@ Version 1.0.0 of this module introduces a breaking change: adding the `disable-l
 In either case, upgrading to module version `v1.0.0` will trigger a recreation of all node pools in the cluster.
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-## Inputs
```

Comment:
Why is this content being removed?

Reply:
I ran `make generate`.  I forgot I need to do `make generate_docs`.